packages/@webex/plugin-meetings/test/unit/spec/locus-info/selfUtils.js

```diff
@@ -251,6 +251,38 @@ describe('plugin-meetings', () => {
           assert.equal(updates.canNotViewTheParticipantListChanged, false);
         });
       });
+
+      describe('localAudioUnmuteRequestedByServer', () => {
+        it('should return localAudioUnmuteRequestedByServer = true when requestedToUnmute = true and lastModifiedRequestedToUnmute timestamps are different', () => {
```

Comment:
i think we should not only be checking that they are different, but that the newer one is also `>` than the existing, so we should add test cases for that.